Loader/Loader.php

```diff
@@ -0,0 +1,21 @@
+<?php
+/**
+ * File containing the Loader class.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace EzSystems\PlatformUIBundle\Loader;
+
+/**
+ * JS and CSS loader.
+ */
+interface Loader
+{
+    /**
+     * @param array $files Array of file names to be loaded.
+     *
+     * @return array Array which elements are JS/CSS content and content type
```

Comment:
I'd really rather have two functions, one to get the content, and another one to get the type.

I know that the type is detected while iterating over the items, but you could very easily extract this part, and only check the first (or last) one.